src/content/getusermedia/audio/js/main.js

```diff
@@ -31,5 +31,4 @@ function handleError(error) {
   console.log('navigator.getUserMedia error: ', error);
 }
 
-navigator.mediaDevices.getUserMedia(constraints).
-    then(handleSuccess).catch(handleError);
+navigator.mediaDevices.getUserMedia(constraints).then(handleSuccess).catch(handleError);
```

Comment:
Same comments as in https://github.com/webrtc/samples/pull/1055 with regards to async/await? But such changes would be nice to comment on in separate PRs because this applies to a lot of files.